server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/ThreadPoolResource.java

```diff
@@ -64,11 +73,17 @@
  */
 public enum ThreadPoolResource implements ResourceDefinition, ThreadPoolDefinition {
 
-    ASYNC_OPERATIONS("async-operations", 25, 25, 1000, 60000),
-    LISTENER("listener", 1, 1, 100000, 60000),
-    REMOTE_COMMAND("remote-command", 25, 25, 100000, 60000),
-    STATE_TRANSFER("state-transfer", 1, 60, 0, 60000),
-    TRANSPORT("transport", 25, 25, 100000, 60000),
+   ASYNC_OPERATIONS("async-operations", getDefaultMinThreads(ASYNC_OPERATIONS_EXECUTOR),
+                    getDefaultThreads(ASYNC_OPERATIONS_EXECUTOR), getDefaultQueueSize(ASYNC_OPERATIONS_EXECUTOR),
+                    60000),
```

Comment:
What about adding a constant for 60000 ?

Reply:
@danberindei ?

Reply:
Ok, I can do that